api/google-manager/services/google-manager.js

```diff
@@ -294,6 +311,16 @@ class MockGoogleManager {
     console.log(`${user_email} joined mock Google Group`);
   }
 
+  async getGroup(groupEmail) {
+    const mockGroup = {
+      id: uuidv4(),
```

Comment:
Why do we generate an UUID here?

Reply:
getGroup returns a google group object in the GoogleManager class. If I don't return an id in the mock one then we'll get an error saying something like `can't get id of undefined`.